tests/queries/0_stateless/02933_paste_join.sql

```diff
@@ -32,6 +32,6 @@ INSERT INTO t2 SELECT number, number FROM numbers(15, 15);
 SELECT * FROM ( SELECT * from t1 ) t1 PASTE JOIN ( SELECT * from t2 ) t2 SETTINGS max_threads = 1;
 SELECT toTypeName(a) FROM (SELECT number as a FROM numbers(11)) t1 PASTE JOIN (select number as a from numbers(10)) t2 SETTINGS join_use_nulls = 1;
 SET max_threads = 2;
+select * from (SELECT number as a FROM numbers_mt(10)) t1 PASTE JOIN (select number as a from numbers(10) ORDER BY a DESC) t2 SETTINGS max_block_size=10;
```

Comment:
Let's add note into docs that in that case result can be nondeterministic (and add an example)